packages/lesswrong/components/posts/PostsPage/PostsPageEventData.tsx

```diff
@@ -17,12 +45,48 @@ const PostsPageEventData = ({classes, post}: {
   
   const locationNode = onlineEvent ? (
     <div className={classes.eventLocation}>Online Event</div>
-  ) : location && <div className={classes.eventLocation}> {location} </div>;
+  ) : (location && <div className={classes.eventLocation}> {location} </div>);
+  
+  let externalLinks;
+  if (post.facebookLink || post.meetupLink || post.website) {
+    externalLinks = (
+      <div className={classes.externalLinks}>
+        {post.facebookLink && (
+          <Tooltip
+            title="See this event on Facebook"
+          >
+            <a href={post.facebookLink} className={classes.externalLink}>
+              <FacebookIcon className={classes.facebookIcon}/>
+            </a>
+          </Tooltip>
+        )}
+        {post.meetupLink && (
+          <Tooltip
+            title="See this event on Meetup"
+          >
+            <a href={post.meetupLink} className={classes.externalLink}>
+              <MeetupIcon className={classes.meetupIcon}/>
+            </a>
+          </Tooltip>
+        )}
+        {post.website && (
+          <Tooltip
+            title="See this event website"
+          >
+            <a href={post.website} className={classes.externalLink}>
+              <LinkIcon className={classes.linkIcon}/>
+            </a>
+          </Tooltip>
+        )}
+      </div>
+    )
+  }
   
   return <Components.Typography variant="body2" className={classes.metadata}>
       <div className={classes.eventTimes}> <Components.EventTime post={post} dense={false} /> </div>
       { locationNode }
       { contactInfo && <div className={classes.eventContact}> Contact: {contactInfo} </div> }
+      { externalLinks }
```

Comment:
Aww, this is a lot easier than what I'm proposing. Still think you should do it my way tho.